app/managers.py

```diff
@@ -0,0 +1,36 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(f"INSERT INTO {self.table_name}"
+                                 f" (first_name, last_name)"
+                                 f" VALUES (?, ?)",
+                                 (first_name, last_name))
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        actors_actor_cursor = self._connection.execute("SELECT * FROM actors")

```

Comment:
```suggestion
        actors_actor_cursor = self._connection.execute(f"SELECT * FROM {self.table_name}")

``` it would be better to use